### DISCUSSION TITLE
"Baseball Cube" v2.0

### DISCUSSION THREAD

**Discussion by mermelmadness** [vasqued2/ha-teamtracker#75] [Show and tell]
Previously I had posted my Baseball Cube, an 8x8x8 cube that displayed the score of my home team's games with color accurate backgrounds for the opponent and the inning.  I have utilized MANY more attributes in Team Tracker to create this:

![image](https://user-images.githubusercontent.com/38080873/222831754-600c6295-52cd-4a17-afb1-6c30a945dcd3.png)

Each of those segments changes with the Team Tracker integration updates, though I learned that when multiple attributes update at once, HA can only send data for ONE over MQTT, so I had to be creative and do active polling for the attributes that can change quickly, like the count and baserunners.

I'm going to expand on it with additional panels once they arrive from Aliexpress.

**Comment by vasqued2** [comment]
Very cool!

**Comment by NRutt2005** [comment]
@mermelmadness Do you have a video laying out the process of how you got this to display on the LED matrix? This is awesome! You love to see how you made this work! Thanks for sharing!

**Reply by mermelmadness** [reply]
I don't.  This took weeks of trial and error to get just right, so no video was recorded.  I did do a write up after the fact, though.

https://docs.google.com/document/d/e/2PACX-1vQoRXeiWJfQLLFJWMy0aDzHYPHIsG2kEdXH1pbNrTZetNeDoELOP7q8-DjHPTnTQBrtyGLlzxL0fPNl/pub

**Reply by NRutt2005** [reply]
Excellent! Thank you for sharing this document! Now I can annoy the wife even more with not just LEDs but Sports! lol

**Reply by mermelmadness** [reply]
My wife likes that I have a "craft" to do.  She has her yarn and glue gun, I have my 3D printer and soldering iron.

**Comment by mermelmadness** [comment]
UPDATE:

Got two panels connected, so now I have live standings, too.

![image](https://user-images.githubusercontent.com/38080873/223896670-0a862070-00d7-43a5-bd36-78c6c32566e4.png)
